README.md

```diff
@@ -1,24 +1,24 @@
 # IBM Cloud Transit Gateway - Terraform Module
 
-This is a collection of modules that make it easier to provision transit gateway and configure multiple connections to it on IBM Cloud Platform:
-* [tg-gateway-connection](modules/tg-gateway-connection)
+With IBM Cloud® Transit Gateway (TGW), you can create a single or multiple transit gateways to connect VPCs together. You can also connect your IBM Cloud classic infrastructure to a Transit Gateway to provide seamless communication with Classic Infrastructure resources. Any new network that you connect to a Transit Gateway is then automatically made available to every other network connected to it. For more info [refer](https://cloud.ibm.com/docs/vmwaresolutions?topic=vmwaresolutions-interconnectivity-tgw-overview)
 
-## Compatibility
+Following diagram illustrates the baisc transit gateway with VMware.
 
-This module is meant for use with Terraform 0.13 (and higher).
+![IBM Cloud Transit Gateway with VMware on VPC](diagrams/transit_gateway.png)
 
-## Usage
+Below are the collection of modules that make it easier to provision transit gateway and configure multiple connections to it on IBM Cloud Platform:
 
-Full examples are in the [examples](./examples/) folder, demonstarte how to use a module through a template:
+* [tg-gateway-connection](modules/tg-gateway-connection)
 
-e.g:
+
+## Usage
 
 ```hcl
 data "ibm_resource_group" "resource_group" {
   name = var.resource_group_name
 }
 
-module "tg-gateway-connection" {
+module "tg_gateway_connection" {
   source = "terraform-ibm-modules/transit-gateway/ibm//modules/tg-gateway-connection"
 
   transit_gateway_name = var.transit_gateway_name
```

Comment:
Would it be possible to replace the variable here with actual sample values - this would gives a better perspective of what is expected from a quick glance.

Reply:
Fixed in latest commit